app/controllers/users_controller.rb

```diff
@@ -5,7 +5,7 @@ def index
   end
 
   def show
-    @user = User.where(id: params[:id])
-    @posts = Post.where(id: params[:id])
+    @user = User.find(params[:id])
+    @posts = @user.posts.includes(:comments)
```

Comment:
[OPTIONAL]
- [ ] I suggest we take this piece of code in the index action. We will get user comments in our views very easily like so `@user.post.comments`